_includes/navbar.html

```diff
@@ -33,8 +33,8 @@
         <img class="navbar-logo" src="{{ relative_root_path }}{% link /assets/img/cp-logo-blue.svg %}" alt="The Carpentries logo" />
       </a>
       {% elsif site.carpentry == "ea" %}
```

Comment:
"ea" stands for escience academy. Please replace it with what is chosen for esmvaltool.